frontend/packages/api_client/src/models/OpenEdXInstanceConfigUpdate.ts

```diff
@@ -68,6 +68,18 @@ export interface OpenEdXInstanceConfigUpdate {
      * @memberof OpenEdXInstanceConfigUpdate
      */
     draftThemeConfig?: ThemeSchema;
+    /**
+     * Your branding to be displayed throughout your instance. It should be 48px tall. If unset, OpenCraft\'s logo will be used.
+     * @type {string}
+     * @memberof OpenEdXInstanceConfigUpdate
+     */
+    readonly logo?: string;
+    /**
+     * This is used as the browser tab icon for your instance\'s pages. If unset, OpenCraft\'s icon will be used.
```

Comment:
I assume there is no size requirement here, unlike the logo image.

Reply:
I'm not sure what you're referring to on this line, but this structure is an auto-generated Typescript `interface` (similar to a model in Django) from the DRF Swagger Schema.
To view the schema, spin up your devstack and go to http://localhost:5000/api/swagger/?format=openapi

Reply:
Sorry if I was not clear. I noticed that in the logo image, you require a 48px tall image and here there is no image dimensions restriction. I wanted to know if there is none or if you had forgotten to add it. Even though this file is auto-generated, I believe you wrote the doc in other file.

Reply:
The feedback on this is enforced by the backend (`/image` endpoint).
If you upload an invalid logo image it should error out.

Reply:
> The feedback on this is enforced by the backend (/image endpoint).
> If you upload an invalid logo image it should error out.

Can we add the validation to check this in the frontend as well so that we don't have to upload the image to the backend to find out issues with the height, if any?